src/github.com/mozilla-services/pushgo/simplepush/config_test.go

```diff
@@ -134,9 +140,14 @@ func TestConfigFile(t *testing.T) {
 	} else {
 		t.Errorf("Pinger type assertion failed: %#v", pinger)
 	}
-	handlers := app.EndpointHandler()
-	if handlers.maxDataLen != 512 {
-		t.Errorf("Wrong maximum data size: got %d; want 512", handlers.maxDataLen)
+	endpointHandler := app.EndpointHandler()
+	if eh, ok := endpointHandler.(*EndpointHandler); ok {
```

Comment:
/me waits for either the MuxEndpointHandler variable (which I'm kinda ambivalent about) or the Functional Utility CallbacK (which might require an HR button press).